clang/lib/CodeGen/CGBuiltin.cpp

```diff
@@ -60,9 +60,14 @@
 #include "llvm/Support/ScopedPrinter.h"
 #include "llvm/TargetParser/AArch64TargetParser.h"
 #include "llvm/TargetParser/X86TargetParser.h"
+#include <algorithm>
 #include <optional>
 #include <sstream>
 
+
+
+#include <iostream>
```

Comment:
iostream is forbidden in LLVM code. If you think the debug prints are useful long-term, use llvm::dbgs(); otherwise, just delete them.

Reply:
changing it to llvm::dbgs() now. will remove it once the patch is ready